backend/apps/github/models/common.py

```
@@ -1,90 +1,104 @@
-            return node.raw_data["node_id"]
-        except UnknownObjectException:
-            pass
+            conversation = Conversation.objects.get(slack_channel_id=channel_id)
+        except Conversation.DoesNotExist:
+            conversation = Conversation(slack_channel_id=channel_id)
+
+        conversation.from_slack(conversation_data, workspace)
+        if save:
+            conversation.save()
+
+        return conversation
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)